tests/test_informationmodel.py

```diff
@@ -121,9 +124,14 @@ def test_to_graph_should_return_publisher() -> None:
     @prefix dcat: <http://www.w3.org/ns/dcat#> .
     @prefix foaf:  <http://xmlns.com/foaf/0.1/> .
 
-    <http://example.com/informationmodels/1>  a dcat:Resource ;
-    dct:publisher <https://example.com/organizations/1> ;
-    .
+    <http://example.com/informationmodels/1> a <http://www.w3.org/ns/dcat#Resource> .
+      <http://example.com/informationmodels/1> dct:publisher
```

Comment:
Du oppretter fortsatt denne ressursen to ganger i samme turtle, Det fungerer helt fint, men legg alle relevante properies inn i samme ressurs.